### DISCUSSION TITLE
DLT viewer plugin blank

### DISCUSSION THREAD

**Discussion by randomdudebunchofnumbers** [COVESA/dlt-viewer#595] [Q&A]
Hi,
I run dlt viewer on macOS Sequoia 15.1.1 (installed from App Store) and I encounter the issue that the dlt viewer plugin window stays blank no matter what I do. The window itself with the different tabs is there but the field where you would normally see the message is empty. I've already tried the following things to resolve the issue:

- enable/disable plugins
- open/close the dlt viewer plugin
- build different versions of dlt viewer from source

Any ideas what I could try next to get dlt viewer plugin to work? Thank you.

**Comment by alexmucde** [comment]
@randomdudebunchofnumbers I have never seen such an issue in Windows or Ubuntu. I am not familar with macOs.

**Comment by alexmucde** [comment]
@hannesa2 Do you have seen something like this?

**Reply by hannesa2** [reply]
Do you mean this ? 
<img width="523" alt="image" src="https://github.com/user-attachments/assets/aa2c3466-de41-405e-9846-c688dc19d901">
